clients/tabby-chat-panel/src/index.ts

```diff
@@ -1,8 +1,11 @@
-import { createThreadFromIframe, createThreadFromInsideIframe } from '@quilted/threads'
+import { createThreadFromIframe } from '@quilted/threads'
```

Comment:
consider get rid of @quilted/threads completely if we go with this approach.

Reply:
offline update: #3474